src/utils.ts

```diff
@@ -10,6 +10,48 @@ export interface Options {
   etherscan: string | undefined;
 }
 
+export function getLink(chain: ChainName, hash: string): string {
+  if (!hash) {
+    return "";
+  }
+  if (chain.includes("ethereum")) {
+    return `https://etherscan.io/tx/${hash}`;
+  } else if (chain.includes("polygon")) {
+    return `https://polygonscan.com/tx/${hash}`;
+  } else if (chain.includes("optimism")) {
+    return `https://optimistic.etherscan.io/tx/${hash}`;
+  } else if (chain.includes("arbitrum")) {
+    return `https://arbiscan.io/tx/${hash}`;
+  }
+  return "";
+}
+
+export function getSignerOnly({
```

Comment:
Not the worst hack, but it is working around a quirk of js-tableland that should be addressed directly.